### DISCUSSION TITLE
Are apparmor and firejail compatible with GNU Guix System?

### DISCUSSION THREAD

**Discussion by crocket** [netblue30/firejail#5057] [Q&A] [answered]
Its system directory layout is different from other operating systems.

It comes with its own experimental unfinished sandboxing system.

Will firejail and apparmor be unable to restrain GNU Guix System?

**Comment by kiasoc5** [comment, marked as answer]
Firejail and apparmor are available on NixOS which also has a different filesystem layout. So it is possible to package apparmor/firejail for Guix. No one has done it yet.
